artemis-chart/templates/postgres-deployment.yaml

```diff
@@ -1,19 +1,21 @@
-apiVersion: extensions/v1beta1
+apiVersion: {{ template "artemis.deployment.apiVersion" . }}
 kind: Deployment
 metadata:
   labels:
-    io.kompose.service: postgres
-    app: {{ .Release.Name }}
+    app: {{ .Release.Name }}-postgres
   name: {{ .Release.Name }}-postgres
 spec:
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}-postgres
   replicas: {{ .Values.replicaCount | default 1 }}
   strategy:
     type: Recreate
   template:
     metadata:
       creationTimestamp: null
       labels:
-        io.kompose.service: postgres
+        app: {{ .Release.Name }}-postgres
```

Comment:
shouldn't also the changes regarding
```
        {{- range $key, $value := .Values.nodeSelector }}
        {{ $key }}: {{ $value }}
        {{- end }}
      {{ toYaml . | nindent 8 }}
      {{- end }}
```
applied here as in the other deployments?